main.rb

```diff
@@ -0,0 +1,48 @@
+morse_code = {
+  '.-' => 'A',
+  '-...' => 'B',
+  '-.-.' => 'C',
+  '-..' => 'D',
+  '.' => 'E',
+  '..-.' => 'F',
+  '--.' => 'G',
+  '....' => 'H',
+  '..' => 'I',
+  '.---' => 'J',
+  '-.-' => 'K',
+  '.-..' => 'L',
+  '--' => 'M',
+  '-.' => 'N',
+  '---' => 'O',
+  '.--.' => 'P',
+  '--.-' => 'Q',
+  '.-.' => 'R',
+  '...' => 'S',
+  '-' => 'T',
+  '..-' => 'U',
+  '...-' => 'V',
+  '.--' => 'W',
+  '-..-' => 'X',
+  '-.--' => 'Y',
+  '--..' => 'Z'
+}
+
+def decode_char(string, morse_code)
+  puts morse_code[string]
+end
+
+def decode_word(morse, morse_code)
+  decoded_chars = morse.split.map { |char| morse_code[char] }
+  decoded_word = decoded_chars.join
+  puts decoded_word
+end
```

Comment:
- [ ] In the `decode_word` method you should use the `decode_char` to translate a single char into a letter, kindly make sure to implement this so you're not repeating your self